api.js

```diff
@@ -108,6 +109,15 @@ Api.prototype._handleExceptions = function (data) {
 };
 
 Api.prototype._handleStats = function (stats) {
+	if (this.hasExclusive && !stats.hasExclusive) {
```

Comment:
Correct, although I'm not super keen on just dropping this info on the floor. I just went for consistency with how non-exclusive tests are dropped within `test-collection`. I'll add a comment to that effect.

That said, I would favor changing the reporting completely as suggested by #471.